internal/v1/handler_blueprints.go

```diff
@@ -546,7 +546,9 @@ func (h *Handlers) ComposeBlueprint(ctx echo.Context, id openapi_types.UUID) err
 	}
 	composeResponses := make([]ComposeResponse, 0, len(blueprint.ImageRequests))
 	clientId := ClientId("api")
-	if ctx.Request().Header.Get("X-ImageBuilder-ui") != "" {
+	if ctx.Request().Header.Get("X-ImageBuilder-ui") == "mcp" {
+		clientId = "mcp"
+	} else {
```

Comment:
This behaviour is actually different now, the default changed from api to ui. 

It should be `else if ctx.Req… != ""`, making the default "api" again.

Reply:
Sorry, thanks - did it in a hurry 🫣